src/reader.rs

```diff
@@ -829,7 +829,7 @@ impl Gcno {
             "{:>9}:{:>5}:Programs:{}",
             "-",
             0,
-            if has_runs { 1 } else { 0 }
+            i32::from(has_runs)
```

Comment:
Same here, unrelated to this PR.

Reply:
i can revert all of those, but the CI lint fails due to this